_tutorials/new-component/create-a-new-component-part-1.md

```diff
@@ -0,0 +1,237 @@
+
+### Output
+
+Now run `obt init` and answer the questions as above. You should see a directory which contains component boilerplate (commonly shared component code we can build on top of).
+
+<figure>
+	<img alt="" src="/assets/images/tutorial-new-component/example-init-output.png" />
+	<figcaption class="o-typography-caption">
+        Example output from the `obt init` command at the time of writing. It shows the questions asked and example responses, as discussed previously.
+	</figcaption>
+</figure>
+
+The structure of our component now looks something like this. In the next section we will start developing our component, and in the process discuss what is in each directory.
+
+```
+o-example
+├── .eslintrc.js
+├── .dependabot
+│   └── config.yml
+├── .github
+│   ├── CODEOWNERS
+│   ├── ISSUE_TEMPLATE.md
+│   └── workflows
+│       └── [a number of .yml files]
+├── .gitignore
+├── .stylelintrc.js
+├── README.md
+├── bower.json
+├── demos
+│   └── src
+│       ├── demo.js
+│       ├── demo.mustache
+│       ├── demo.scss
+│       └── pa11y.mustache
+├── main.js
+├── main.scss
+├── origami.json
+├── package.json
+├── src
+│   ├── js
+│   │   └── example.js
+│   └── scss
+│       ├── _brand.scss
+│       └── _variables.scss
+└── test
+    ├── js
+    │   ├── example.test.js
+    │   └── helpers
+    │       └── fixtures.js
+    └── scss
+        ├── _main.test.scss
+        └── index.test.scss
+```
+
+## Source Control
+
+All Origami components reside in a [git](https://git-scm.com/) repository with the same name as the component. It's required that component repositories are stored remotely in one of our [github.com](https://github.com/) organisations, for example the [Financial-Times](https://github.com/Financial-Times/) organisation. There are more details about [source control in the origami specification](/spec/v1/components/#source-control).
+
+Create a new git repository by running `git init`, and commit the boilerplate as an initial commit. For example:
+
+<pre><code class="o-syntax-highlight--bash">git init
+git add --all
+git commit -m 'o-example component boilerplate'</code></pre>
+
+You may then push this to your remote github.com repository, under the `Financial-Times` organisation or another [supported Financial Times organisation](/spec/v1/components/#source-control).
+
+_Note: we're assuming existing git knowledge here, so please feel free to contact the Origami team if you get stuck at any point._
+
+You might have noticed a `.github` directory. At the time of writing, the `.github` directory itself is not part of the [file structure specification](/spec/v1/components/#files-and-folder-structure). If we deleted `.github` the component would still be an Origami component. But this directory gives us some nice features including:
```

Comment:
We should think about making the `.github` directory an optional feature in the create-origami-component generator since Origami is currently the only team which uses GitHub Actions and most other teams are using CircleCI.

Reply:
I don't think we should do that unless we're going to provide circleci config for our users so occ etc is setup correctly. And we should check its something users actually want before creating that